lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -225,9 +225,18 @@ SPIRVToLLVMDbgTran::transTypeArray(const SPIRVExtInst *DebugInst) {
     if (!getDbgInst<SPIRVDebug::DebugInfoNone>(Ops[I])) {
```

Comment:
May we use `if .. else if ..` instead of `continue` at the end of each if block?

Reply:
I am afraid that in the future there may be too many different "else-if" blocks, it will be difficult to fit into 80 characters

Reply:
Ok, this is just my opinion, but I think that a code is less error prone and easier to understand when `if .. else if` is used.
Fitting into 80 characters is not so important, IMHO.